.gitignore

```diff
@@ -2,3 +2,14 @@
 *.o
 *.swp
 *.swo
+*.pb.h
+*.pb.cc
+*.pb.go
+*.so
+*.cmd
+*.exe
+*.bin
+app1_data/
+app2_data/
+simpleserver
+provisioning/
```

Comment:
The above look like valid r.e's to exclude some generated files. With these rules, now we are reduced to the following shortened output from `git status`:

```
sdb-fdb-build:[96] $ git status
On branch agurajada/fix-mak-files-rm
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	sample_apps/simple_app/policy_key.cc
	sev-snp-simulator/Module.symvers
	sev-snp-simulator/modules.order
	sev-snp-simulator/sevnull.ko
	sev-snp-simulator/sevnull.mod

nothing added to commit but untracked files present (use "git add" to track)
```
I don't know if I should add specific rules for these files, or let it go. I suspect these are all also generated files, so could be git-ignore'ed.

Please advise.